students/HABTAMU/lesson09/assignment/test_database.py

```diff
@@ -0,0 +1,26 @@
+from pymongo import MongoClient
+import csv
+import database as d_base
+import pytest
+import pprint
+
+client = MongoClient('localhost', '27017')
+client.drop_database('HP_Norton')
+
+
+def drop_db():
+    client = MongoClient('localhost', '27017')
+    client.drop_database('HP_Norton')
+
+def create_db():
+    myclient = MongoClient("mongodb://localhost:27017/")
+    db = myclient.HP_Norton
+
+def create_collections():
+    create_db()
+    # Customers collection
+    customers = db["customers"]
+    # Products collection
+    products = db["products"]
+    # Rentals collection
+    rentals = db["rentals"]
```

Comment:
This file looks like it doesn't have tests in it. You should only name things test_  if they actually have tests in it. Less confusing that way. :)

Reply:
I started doing test module and was distracted and/ or haven't got enough time latter on, will try to have test on a brake, and thank you